dipper/sources/PostgreSQLSource.py

```diff
@@ -135,20 +131,20 @@ def fetch_query_from_pgdb(self, qname, query, con, cxn, limit=None, force=False)
         outputquery = \
             "COPY ({0}) TO STDOUT WITH DELIMITER AS '\t' CSV HEADER".format(query)
 
-        with open(outfile, 'w') as f:
-            cur.copy_expert(outputquery, f)
+        with open(outfile, 'w') as tsvfile:
+            cur.copy_expert(outputquery, tsvfile)
         # Regenerate row count to check integrity
         filerowcount = self.file_len(outfile)
         if (filerowcount-1) < tablerowcount:
             raise Exception(
-                    "Download from %s failed, %s != %s", cxn['host'] + ':' +
-                    cxn['database'], (filerowcount-1), tablerowcount)
-        elif (filerowcount-1) > tablerowcount:
-                LOG.warning(
-                    "Fetched from %s more rows in file (%s) than reported in count(%s)",
-                    cxn['host'] + ':'+cxn['database'], (filerowcount-1), tablerowcount)
-
-    # TODO generalize this to a set of utils
+                "Download from {} failed, {} != {}"
+                .format(cxn['host'] + ':' + cxn['database'],
+                        (filerowcount-1), tablerowcount))
+        if (filerowcount-1) > tablerowcount:
+            LOG.warning(
```

Comment:
Getting an indentation error here: 

PostgreSQLSource.py:144:0: C0330: Wrong hanging indentation (remove 4 spaces).
                    "Download from %s failed, %s != %s", cxn['host'] + ':' +
                |   ^ (bad-continuation)
PostgreSQLSource.py:145:0: C0330: Wrong hanging indentation (remove 4 spaces).
                    cxn['database'], (filerowcount-1), tablerowcount)
                |   ^ (bad-continuation)

Reply:
odd, I'm not getting this warning, are you running it on 44a9c5b?

Reply:
Yep, sorry, needed to do a git pull, I was on [this previous commit](https://github.com/monarch-initiative/dipper/pull/760/commits/0e6f91e25c1a794c2f761257e4d2f99726404512)